src/frame.test.tsx

```diff
@@ -93,7 +101,7 @@ describe("adding elements to frames", () => {
       await testElements(
         "rectangle",
         ["text", "rectangle", "frame"],
-        ["text", "rectangle", "frame"],
+        ["rectangle", "text", "frame"],
```

Comment:
Maybe I broke something with this, but why would the bound text element be expected to be rendered below its container?